ekan0ra/queue.py

```diff
@@ -25,21 +25,22 @@ def enqueue(self, nick):
         """Add nick to queue."""
         self.append(nick)
 
-    def dequeue(self, nick):
+    def dequeue(self, nick, all=False):
         """
-        Remove first item matching `nick` in queue.
+        Remove every occurrence of `nick` from queue.
```

Comment:
@rahuldecoded What about the above comment?

Not to worry, I'll fix that.
Good job so far.